Makefile.configure

```diff
@@ -15,11 +15,12 @@ PROGRAM-freestanding += int main() {}
 CCFLAGS-freestanding = $(CFLAGS)
 
 PROGRAM-cjson  = \#include <stdlib.h>\n
+PROGRAM-cjson += \#include <math.h>\n
 PROGRAM-cjson += \#include <cjson/cJSON.h>\n
-PROGRAM-cjson += int main() { return cJSON_False; }
-CCFLAGS-cjson = -I$(dir $(MAKEFILE))src
+PROGRAM-cjson += int main() { double d = NAN; return cJSON_False; }
```

Comment:
Why did you need to add `double d = NAN;` here?

Reply:
I didn't realise I had. I have a vague memory of this being necessary to not enable cJSON on FP-less systems, but I don't recall the details.

Reply:
Ok, I don't see this being an issue since it looks like the value is unused. If it was added to enable cJSON for unusual system the odd code makes sense.